handlers/services.go

```diff
@@ -111,7 +111,7 @@ func ServiceDetails(w http.ResponseWriter, r *http.Request) {
 	namespace := params["namespace"]
 	service := params["service"]
 	queryTime := util.Clock.Now()
-	rateInterval, err = adjustRateInterval(r.Context(), business, namespace, rateInterval, queryTime)
+	rateInterval, err = adjustRateInterval(r.Context(), business, cluster, namespace, rateInterval, queryTime)
```

Comment:
Should we use the cluster here? 
When we choose a list of namespaces from different cluster, I receive a 404 error: 

![image](https://user-images.githubusercontent.com/49480155/234835364-e7f7a008-5730-4ee3-8aa9-b97598d1ed26.png)

Reply:
I see the issue now. Agree, this was wrong asumption, because with empty cluster it looks namespaces through all clusters, reverting this.